package.json

```diff
@@ -9,7 +9,7 @@
     "deps": "make -C ./pybind11/ && tools/install-python.js",
     "preinstall": "npm install @mapbox/node-pre-gyp",
     "install": "node-pre-gyp install --fallback-to-build=false || npm run build",
-    "pretest": "npm run lint && ./.miniconda/bin/pip install numpy==1.17.4",
+    "pretest": "npm run lint && ./tools/bip.js install numpy==1.17.4",
```

Comment:
As per https://github.com/imgcook/boa/issues/10#issuecomment-963833617

Reply:
Without this change we get the following error scattered throughout the tests:

``` 
  TypeError: ModuleNotFoundError: No module named 'numpy'
```

Reply:
There is a discussion that took place in #10. How would you like me to proceed?

Reply:
@yorkie I've just looked into this further. 

If `numpy` is present in `.miniconda/lib/python3.7/site-packages` when we **build** then it is copied across to `binding/v6/lib/python3.7/site-packages`, if it is not present then running `./.miniconda/bin/pip` will not make the package available to `boa`. It will only install the package into `.miniconda/lib/python3.7/site-packages`, this is the reason that we have to use the `bip` tool to install `numpy` directly into `binding/v6/lib/python3.7/site-packages`. 

This, therefore, confirms that the package is working as expected as users will also be using `bip` to install further packages that they need.